src/_includes/soft_mod_hubspot_cheat_sheet_banner_cta.liquid

```diff
@@ -0,0 +1,5 @@
+{%- capture hubspot-id -%} 
+  {%- if site.lang == 'es' -%}041e3783-91f3-4bc1-8926-7376fa4faae0{%- else -%}06e1ce72-a796-4e1c-8ff1-1b9c7a67fe57{%- endif -%}
+{%- endcapture -%}
+
+{%- include hubspot_cta.liquid class="hubspot-in-page-banner" hubspot-id=hubspot-id -%}
```

Comment:
Once the changes above have been made this file becomes redundant and can be deleted.

Reply:
File has been deleted